consensus/ledger_validator/ledger_validator.c

```diff
@@ -150,13 +149,12 @@ static retcode_t get_latest_diff(ledger_validator_t *const lv,
     goto done;
   }
 
-  // Add null hash to the set of visited hashes
-  if ((hash_set_elem = malloc(sizeof(hash_set_t))) == NULL) {
-    ret = RC_LEDGER_VALIDATOR_OOM;
-    goto done;
+  {
+    flex_trit_t null_hash[FLEX_TRIT_SIZE_243] = {FLEX_TRIT_NULL_VALUE};
```

Comment:
does that initialize the entire array or just sets the first element?

Reply:
Whoops